### ISSUE TITLE
Panel crashes on typing in "Apps" search

### ISSUE THREAD

**Issue by pushqrdx** [jmanc3/winbar#4]
if i change the tab to applications in the search menu and start typing, panel crashes with out of range exception

```
terminate called after throwing an instance of 'std::out_of_range'
  what():  basic_string::replace: __pos (which is 1) > this->size() (which is 0)
Aborted
```

**Comment by jmanc3** [comment]
Just pushed a fix! This issue was such headache to figure out but it's finally fixed in 65e3143.

**Comment by pushqrdx** [comment]
@jmanc3 sweet! works so much better that most other panels out there, keep up the good work :D
